core/storage/topic/gae_models.py

```diff
@@ -484,6 +477,14 @@ def verify_model_user_ids_exist(self):
             user_ids, include_deleted=True)
         return all(model is not None for model in user_settings_models)
 
+    def compute_snapshot(self):
```

Comment:
Why do we need this for this model and not the other two?

Also please compare the TopicModel code to the others in general -- I would expect the changes to the three models to be broadly similar.

Reply:
Done.